src/escalus_connection.erl

```diff
@@ -238,9 +239,14 @@ get_stanza_safe(Client, Timeout) ->
 get_stanza_safe(Client, Timeout, Pred) ->
     receive_stanza(Client, #{timeout => Timeout, with_metadata => true, safe => true, pred => Pred}).
 
--spec wait(client(), timeout()) -> ok.
+-spec wait(client(), timeout()) -> no_return().
 wait(Client, Timeout) ->
-    {error, timeout} = receive_stanza(Client, #{timeout => Timeout, pred => fun(_) -> false end}),
+    receive_stanza(Client, default_recv_opts(Timeout)).
+
+-spec wait_safely(client(), timeout()) -> ok.
+wait_safely(Client, Timeout) ->
```

Comment:
@mkacper @erszcz do you still need the `wait_safely` function. I can see that current master branch uses `safe => true` by default in `wait` function. See: https://github.com/esl/escalus/blob/66e25b838d936bb160723259b096b800ccb6f88e/src/escalus_connection.erl#L249

Reply:
The function is used in an Amoc scenario that we've used in some load tests. We, probably, would like to be able to check what Amoc and Escalus versions were used in these load tests.  Any ideas how we can proceed with that?

Reply:
How many such scenarios do you have? Maybe it'd be better to modify them to use the `wait` function which is by default safe in master now?

Reply:
@michalwski I've removed that function. PR is ready to be merged.

Reply:
@michalwski The functionality of `safe => true` is fine and it was made the default upon our report. This branch contained a fix we proceeded with, before the default was changed.

What I'd like to make sure is that the original commit pointed at by the test is accessible (@mkacper can you find the line where it's defined in the load test repo and link here?), not necessarily that this code is merged with the `wait_safely` function present.

Reply:
@erszcz An Escalus version that was used in our load tests is pointed in Amoc: https://github.com/esl/amoc/blob/archived/aws-helpers/rebar.config#L14 which version is, in turn, pointed in the load test repo (I'm not linking the code here as the repo is private).

The Escalus version (commit) is backed up here: https://github.com/esl/escalus/tree/archived/escalus-aws-hotfixes.

Reply:
@mkacper Awesome, thank you!